server_admin/topics/groups.adoc

```diff
@@ -1,38 +1,50 @@
+
 == Groups
 
-Groups in {project_name} allow you to manage a common set of attributes and role mappings for a set of users.
-Users can be members of zero or more groups.
-Users inherit the attributes and role mappings assigned to each group.  To manage groups go to the `Groups` left menu
-item.
+Groups in {project_name} manage a common set of attributes and role mappings for each user. Users can be members of any number of groups and inherit the attributes and role mappings assigned to each group.
+
+To manage groups, click `Groups` in the left menu.
 
 .Groups
 image:{project_images}/groups.png[]
 
-Groups are hierarchical.
-A group can have many subgroups, but a group can only have one parent.
-Subgroups inherit the attributes and role mappings from the parent.
-This applies to the user as well.
-So, if you have a parent group and a child group and a user that only belongs to the child group, the user inherits the attributes and role mappings of both the parent and child.
-In this example, we have a top level `Sales` group and a child `North America` subgroup.  To add a group, click on the
-parent you want to add a new child to and click `New` button.  Select the `Groups` icon in the tree to make a top-level group.
-Entering in a group name in the `Create Group` screen and hitting `Save` will bring you to the individual group management page.
+Groups are hierarchical. A group can have multiple subgroups but a group can have only one parent. Subgroups inherit the attributes and role mappings from their parent. Users inherit the attributes and role mappings from their parent as well.
+
+If you have a parent group and a child group, and a user that belongs only to the child group, the user in the child group inherits the attributes and role mappings of both the parent group and the child group.
+
+In the following example, there is a top level `Sales` group and a child `North America` subgroup.  
```

Comment:
How about "The following example includes a top level..."
Avoiding "there is" makes it more concise and results in active voice.